code/modules/vehicles/wheelchair.dm

```diff
@@ -0,0 +1,126 @@
+/obj/vehicle/ridden/wheelchair
+	name = "wheelchair"
+	desc = "A chair with big wheels. It looks like you can move in this on your own."
+	icon = 'icons/obj/vehicles.dmi'
+	icon_state = "wheelchair"
+	layer = OBJ_LAYER
+	max_integrity = 100
+	armor = list("melee" = 10, "bullet" = 10, "laser" = 10, "energy" = 0, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 20, "acid" = 30)
+	legs_required = 0
+	arms_requires = 0
```

Comment:
Wait you can drive it while you are a nugget?

Reply:
That seems like a consistency issue but makes sense if your being pushed by someone.

Reply:
Yeah, you should always move when being pushed, but you shouldn't be able to move it yourself without arms

Reply:
Read further down in the code.

Reply:
True, my mistake then